apps/src/templates/curriculumCatalog/noSectionsToAssignDialogs.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
+import Typography from '@cdo/apps/componentLibrary/typography';
+import style from './no_sections_to_assign_dialog.module.scss';
+
+export const SignInToAssignSectionsDialog = ({onClose}) => (
+  <NoSectionsToAssignBaseDialog
+    headerText={i18n.signInToAssign()}
+    helpText={i18n.signInToAssignHelpText()}
+    onClose={onClose}
+    buttonText={i18n.signInOrCreateAccount()}
+    href={`/users/sign_in?user_return_to=${location.pathname}`}
+  />
+);
+
+SignInToAssignSectionsDialog.propTypes = {
+  onClose: PropTypes.func.isRequired,
+};
+
+const NoSectionsToAssignBaseDialog = ({
+  headerText,
+  helpText,
+  onClose,
+  buttonText,
+  href,
+}) => {
+  return (
+    <AccessibleDialog onClose={onClose} className={style.dialogueContainer}>
+      <Typography semanticTag="h3" tabIndex="0">
+        {headerText}
+      </Typography>
+      <hr />
+      <Typography semanticTag="span" visualAppearance="body-one">
```

Comment:
This should be a `p` tag instead of a `span`.

Reply:
Oh clever. That makes sense!